### ISSUE TITLE
'data' ==> 'indices_data.py' ==> impossible to get 'world' indices like 'TRCCRB'

### ISSUE THREAD

**Issue by AF20** [alvarobartt/investpy#272]
File "C:\Users\Andrea\AppData\Local\Programs\Python\Python36\lib\site-packages\investpy\indices.py", **line 498, in get_index_historical_data**
    **raise RuntimeError("ERR#0034: country " + country.lower() + " not found, check if it is correct.")**

**Comment by alvarobartt** [comment]
Hi @AF20! AFAIK the world indices are not included there, please check the following file [investpy/resources/indices.csv](https://github.com/alvarobartt/investpy/blob/master/investpy/resources/indices.csv) and let me know if there's any value on the country column with "world", if not, please could you provide the Investing.com URL with the information you are looking for?

Also notice that there's a function named `investpy.search_quotes()` that does that work for you, so that you just need to use the search engine to retrieve the data from the index you want to. 🤗 

More information available at: https://github.com/alvarobartt/investpy/wiki/investpy.search_quotes()

**Comment by AF20** [comment]
Hi @alvarobartt, this is the investing url of the instrument: https://www.investing.com/indices/thomson-reuters---jefferies-crb

In the file investpy/resources/indices.csv there is this instrument:
![image](https://user-images.githubusercontent.com/24624333/106396150-762a5800-6406-11eb-8d3a-f96f1aaf65e1.png)

I don't understand how to use the function, I tried:
search_results = investpy.search_quotes(text='CRB',
                                        products=['indices'],
                                        countries=['world'],
                                        n_results=10)
but I get the error:

in search_quotes
    raise ValueError('ERR#0129: countries filtering parameter possible values are: \"' + ', '.join(cst.COUNTRY_FILTERS.keys()) + '\".')
ValueError: ERR#0129: countries filtering parameter possible values are: "andorra, argentina, australia, austria, [...], turkey, uganda, ukraine, dubai, united kingdom, united states, venezuela, vietnam, zambia, zimbabwe".

**Comment by alvarobartt** [comment]
Yeah, it seems that world is not included among all the possibilities... I'll check this now! :+1:

**Comment by alvarobartt** [comment]
As far as I can see, you are not using the proper function... I mean you can either use the search engine with the function `investpy.search_quotes()` or you can also use the index retrieval functions as `investpy.get_index_recent_data()` or `investpy.get_index_historical_data()`! 👍🏻 

For your concrete example it should be like:

```python
data = investpy.get_index_recent_data(index='TR/CC CRB')
```

Or the search function without using the country filtering:

```python
search_results = investpy.search_quotes(text='CRB',
                                                              products=['indices'],
                                                              n_results=10)
```

__Note that you need to use the index name on the `investpy.get_index_recent_data()` and on the `investpy.search_quotes()` you can use any type of input you want to!__

Hope this was clear enough for you! 🤗

**Comment by AF20** [comment]
data = investpy.get_index_recent_data(index='TR/CC CRB')
==> this is not working like this ==> TypeError: get_index_recent_data() missing 1 required positional argument: 'country'.

data = investpy.get_index_recent_data(index='TR/CC CRB', country = 'world')
==> raise RuntimeError("ERR#0034: country " + country.lower() + " not found, check if it is correct.")



search_results = investpy.search_quotes(text='CRB',
                                        products=['indices'],
                                        n_results=10)
finds 2 results: CROBEX, CROBEX10 which are stock indices of Croazia. 

The CRB (TR/CC CRB) I am searching is a commodity index. https://en.wikipedia.org/wiki/Refinitiv/CoreCommodity_CRB_Index

**Comment by alvarobartt** [comment]
Yeah I messed up in the first code block, I forgot you were supposed to use the `country` parameter too 😩 Anyway, the search function should work... Let me check it later so as to see if the commodity index you want to retrieve can be retrieved using the search function!

Anyway, later this week `investpy v1.0.3` will be released with some bug-fixes and improvements mainly in `investpy.search_quotes()` and `investpy.economic_calendar()` functions. So stay tuned! 👍🏻

**Comment by alvarobartt** [comment]
Hi @AF20, sorry for not reporting this earlier, but this issue was already solved in the `v1.0.3` release, that you can install with:

```
pip install investpy --udgrade
```

Here's the piece of code that you should be using:

```python
import investpy

data = investpy.get_index_recent_data(index='TR/CC CRB', country="world")
print(data.head())
```

Hope this was useful to you! 👍🏻
